ddsrouter_test/compose/test_cases/stress/multi_routers_cloud/compose.yml

```diff
@@ -0,0 +1,284 @@
+# Test description:
+#   This test simulates a busy network:
+#   If we set up four routers, 2 edges and 2 clouds (the cloud router will forward the
```

Comment:
NIT: Please replace typed out numbers like `four` and `twelve` with their actual numbers (4 and 12). It's confusing to read `four router, 2 edges and 2 clouds`.

Reply:
There's still `three best-effort subscribers on three different topics`.